galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -353,3 +353,15 @@ messages:
     level: Warning
     description: "Virtual IP addresses are required for ports serving TCP (or unset) protocol"
     template: "ServiceEntry addresses are required for this protocol."
+
+  - name: "IngressRouteRulesNotAffected"
+    code: IST0135
+    level: Warning
+    description: "Virtualservice not affected Ingress route"
+    template: "Subset configuration in %s not affected ingress route in %s."
```

Comment:
Perhaps we should include the subset name?  The virtual service name should already appear, as the message will be connected to it by the call to `ctx.Report()`, so we probably don't need it here.

Reply:
not sure, since a virtual service can have multiple subset configuration.

Reply:
Maybe "Subset in virtual service %q has no effect on ingress gateway %q requests"

Reply:
this is done @esnible